src/plugins/home/public/application/components/solutions_section/solution_title.test.tsx

```diff
@@ -42,121 +43,51 @@ const solutionEntry = {
   order: 1,
 };
 
+const mockTitle = 'Page Title';
+const makeProps = () => ({
+  title: solutionEntry.title,
+  subtitle: solutionEntry.subtitle,
+  iconType: solutionEntry.icon,
+  branding: {
+    applicationTitle: mockTitle,
+  },
+  logos: getLogosMock.default,
+});
+
 describe('SolutionTitle ', () => {
-  describe('in default mode', () => {
-    test('renders the title section of the solution panel', () => {
-      const branding = {
-        darkMode: false,
-        mark: {
-          defaultUrl: '/defaultModeUrl',
-          darkModeUrl: '/darkModeUrl',
-        },
-        applicationTitle: 'custom title',
-      };
-      const component = shallow(
-        <SolutionTitle
-          title={solutionEntry.title}
-          subtitle={solutionEntry.subtitle}
-          iconType={solutionEntry.icon}
-          branding={branding}
-        />
-      );
-      expect(component).toMatchSnapshot();
-    });
+  it('renders correctly by default', () => {
+    const props = {
+      ...makeProps(),
+    };
```

Comment:
Unnecessary memory io?

```suggestion
    const props = makeProps();
```

Reply:
it sure is but as a pattern across the tests, purely for ease of future editing, it shouldn't be a problem.